db/migrate/20191017054450_add_would_auto_approve_to_measurement_imports.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class AddWouldAutoApproveToMeasurementImports < ActiveRecord::Migration
+  def up
+    add_column :measurement_imports, :would_auto_approve, :boolean, default: false
```

Comment:
Adding a column with a default value to an existing table might cause the entire table to be rewritten. `measurement_imports` is quite big, I think. So, you might want to do like:

```ruby
add_column :measurement_imports, :would_auto_approve, :boolean,
change_column_default :measurement_imports, :would_auto_approve, false)
```

Reply:
is it fine to edit migrate files that had already been migrated?

Reply:
@ZacharyCChang0828 yeah, especially when your local db is the only place it's been applied.

`rake db:migrate:redo` will redo your last one. Details here https://edgeguides.rubyonrails.org/active_record_migrations.html#rolling-back